apps/backend/src/models/questionTypes/TechniquePicker.ts

```diff
@@ -76,9 +76,12 @@ export const techniquePickerDefinition: Question<DataType.TECHNIQUE_PICKER> = {
 
       return {
         ...config,
-        techniques: uniqueTechniques.map(
-          (technique) => new TechniqueOptionClass(technique.id, technique.name)
-        ),
+        techniques: uniqueTechniques
+          .map(
+            (technique) =>
+              new TechniqueOptionClass(technique.id, technique.name)
+          )
+          .sort((a, b) => (a.name > b.name ? 1 : -1)),
```

Comment:
```suggestion
          .sort((a, b) => (a.name > b.name ? 1 : -1)),
```

Will it be a good idea to lowercase the names before sorting to improve the accuracy of sorting?